package/Dockerfile-windows.agent

```diff
@@ -2,15 +2,15 @@ ARG SERVERCORE_VERSION
 FROM mcr.microsoft.com/windows/servercore:${SERVERCORE_VERSION}
 ARG RELEASES
 ARG VERSION
-USER ContainerUser
```

Comment:
Was this incompatible with `Invoke-WebRequest`? We added it as part of #2042.

Reply:
I see. Anyway, this is most likely a short-term fix as we may well end up cross-building Windows-based images from Linux hosts, meaning that download/unzip steps would happen from another image. We could then (but only then) keep `USER ContainerUser`.

TL;DR: this is fine, at least as a short-term fix. My first comment may have been a bit too hasty :þ

Reply:
Maybe we can move it to the bottom? So the powershell commands are still allowed to write to `C:\`.. :shrug: